temp/encoded-file

```diff
@@ -0,0 +1,4 @@
+IyEvb3B0L3B1cHBldGxhYnMvcHVwcGV0L2Jpbi9ydWJ5CgpyZXF1aXJlICdqc29uJwoKYXJncyA9
```

Comment:
Lines 2-4 appear to be junk when I try to decode them.

Reply:
I think I may have appended instead of rewritten....